portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path

```diff
@@ -62,7 +62,7 @@
 </tr>
 <tr>
 	<td>DIALOG_BOX</td>
-	<td>//div[contains(@class,'modal-header')]/h3</td>
+	<td>//div[contains(@class,'modal-title')]</td>
```

Comment:
This path is used in a lot of tests/macros. Are you certain changing this won't affect other tests?

Also, this path isn't technically an Iframe so maybe we can move it out or get rid of it eventually. There might be something in Modal.path that can be used instead.

Reply:
The path is working in other tests as well. These were reproduced and resulted in success.

```
CPDynamicdatalists#AddFormTemplate
PGDynamicdatalists#AddFormTemplate
DepotWorkflow#CanApproveWCArticleByContentReviewer
DepotWorkflow#CanRejectWCArticleByContentReviewer
MyWorkflowTasks#CanAssignTaskToOtherUser
KaleoformsUsecase#AssignRolePermissionsCompleteFormPG
```

Reply:
And yes, there is a path in Modal.path doing the same thing. I can create another pr to do a refactoring so that all tests use the path in modal.path.

Reply:
@victorhcunha great! thanks for checking. The refactor is a low priority, but still nice to have a ticket for and work on eventually. If there are higher priority tasks, then the refactor can wait.